src/input.rs

```diff
@@ -338,13 +337,26 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
         self.ctx.write_to_pty(msg.into_bytes());
     }
 
-    pub fn mouse_report(&mut self, button: u8, state: ElementState) {
-        if self.ctx.terminal_mode().contains(mode::TermMode::SGR_MOUSE) {
-            self.sgr_mouse_report(button, state);
+    pub fn mouse_report(&mut self, button: u8, state: ElementState, modifiers: ModifiersState) {
+        // Calculate modifiers value
+        let mut mods = 0;
+        if modifiers.shift {
+            mods += 4;
+        }
+        if modifiers.logo {
```

Comment:
The protocol specifies 8 is Meta (Alt, in Glutin terms) — Glutin's Logo modifier is X11's Super or Hyper.

Reply:
You're right, alt probably makes more sense here. Meta is just super for me :P.